app/lib/shared/search_service.dart

```diff
@@ -73,46 +75,51 @@ class PackageDocument extends Object with _$PackageDocumentSerializerMixin {
 
 class PackageQuery {
   final String text;
-  final String type;
+  final PlatformPredicate platform;
```

Comment:
To avoid confusion, we should name variables of type `PlatformPredicate` also `platformPredicate` (also at other places)